src/components/camera/camera.tsx

```diff
@@ -75,7 +73,7 @@ export class CameraPWA {
     await this.initCamera();
   }
 
-  componentDidUnload() {
+  disconnectedCallback() {
```

Comment:
`componentDidUnload` was removed in favor of `disconnectedCallback` in Stencil v2 [(source)](https://github.com/ionic-team/stencil/blob/main/BREAKING_CHANGES.md#use-disconnectedcallback-instead-of-componentdidunload).